addon/components/bs4/bs-carousel.js

```diff
@@ -1,8 +1,12 @@
 import Carousel from 'ember-bootstrap/components/base/bs-carousel';
+import { equal, readOnly } from '@ember/object/computed';
```

Comment:
`readOnly` is not used, which makes ESLint fail.

Reply:
Ah, whoops. Just pushed a fix.